packages/orbit-components/src/ListChoice/ListChoice.stories.tsx

```diff
@@ -64,7 +64,6 @@ export const MultipleChoices: Story = {
     selected: false,
     disabled: false,
     tabIndex: 0,
```

Comment:
just small nitpick (and out of your PR), could you set the range for `tabIndex` to have the possible values clear to users?

something like this:
```
argTypes: { tabIndex: { control: { type: 'number', min:-1, max:0 }}
```

Reply:
To me, tabIndex shouldn't even be in the controls… Storybook controls should be more about of what we see. But won't reinvent this now. However, tabIndex can have any number users want. Although it is recommended to only use `-1` or `0`, there is no actual limitation to other numbers, so I don't feel our storybook should be enforcing something that isn't actually enforced

Reply:
ok, my idea was to align the information according the "best practices" of the `tabIndex`.